MediaBrowser.Model/Lyrics/TimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// TimeTag model.
+/// </summary>
+public class TimeTag
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TimeTag"/> class.
+    /// </summary>
+    /// <param name="position">The character index of the lyric.</param>
+    /// <param name="timestamp">The timestamp the lyric is synced to in ticks.</param>
+    public TimeTag(int position, long timestamp)
```

Comment:
The Position refers to the start index but there is no end index or size/width

Reply:
Would it be best for me to add an end index or a size/width entry? The LRC library only reports times that the TimeTags are placed in there, so I'm pretty sure I would need to get the start of the next index to get either the size or the end index.

Reply:
From a client perspective I want to know the start & end position of each cue to apply styling to it. So an ""Width"/"Length"/"Size" property would be useful to have, that way if we ever add support for lyric formats that have cues for some sections but not all in a line we don't need to change the API again.